.gitignore

```diff
@@ -10,7 +10,6 @@ node_modules
 .flutter-plugins
 .packages
 .dart_tool
-# pubspec.lock
```

Comment:
Hi @legalcodes @johnpryan et all. Why was it decided that we'd start committing the `pubspec.lock` files contained in the repo?

Reply:
Hey all, I created a fresh issue to discuss this (#3107) rather than having only a comment in an old PR. (Sorry for the email notification noise.)

Reply:
Oups, my bad, I wasn't looking at this carefully. You just deleted a commented out line! I'm the one who commented it out in #1237 last year. I didn't realize that it had been that long. I thought that it was a change that had been done in the last 6 months or so. Apologies! 

FYI, I'll still go ahead a remove the `pubspec.lock` files.